style.css

```diff
@@ -0,0 +1,235 @@
+@import url("https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@500&family=Roboto+Mono:wght@500&family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgba(200, 200, 200, 0.1);
+  font-family: Roboto, Arial, “Sans Serif”, Noto, “YT Sans”, “Roboto Mono”;
+}
+
+/* .video { */
+/* margin-left: initial; */
+/* padding-left: 100px; */
+/* padding-top: 30px; */
+/* } */
+
+.icon-spacing {
+  padding: 30px 15px;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+.user-comments {
+  display: grid;
+}
+.user-avatar {
+  grid-template-columns: 20%;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.grid-container {
+  display: grid;
+  /* grid-template-columns: 3fr 0.05fr; */
+  justify-content: end;
+}
+
+nav {
+  align-items: center;
+  position: fixed;
+  top: 0%;
+  width: 100%;
+  height: 50px;
+  background-color: #ffffff;
+  justify-content: space-between;
+  z-index: 1;
+}
+
+#left-navbar {
+  justify-content: space-evenly;
+  width: 12%;
+  align-items: center;
+  margin-left: 15px;
+}
+
+#mid-navbar {
+  align-self: center;
+  margin-right: 20px;
+  width: 50%;
+  justify-content: center;
+}
+
+#search-bar {
+  background-color: white;
+  width: 85%;
+  padding: 5px 10px;
+  border: 1px solid silver;
+  border-radius: 3px 0px 0px 3px;
+  font-size: medium;
+}
+
+#search-button {
+  border: 1px solid gray;
+  padding: 10px 30px;
+  border-radius: 0px 3px 3px 0px;
+}
+
+#right-navbar {
+  width: 150px;
+  justify-content: space-between;
+  margin-right: 40px;
+}
+
+.main-video {
+  margin-left: initial;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.flex-horizontal {
+  flex-direction: row;
+}
+
+.upcoming {
+  font-family: Roboto, Arial, “Sans Serif”, Noto, “YT Sans”, “Roboto Mono”;
+  font-size: 16px;
+  color: #030303;
+}
+.input {
+  width: 550px;
+  position: relative;
+  left: 219px;
+}
+
+.container {
+  position: relative;
+  top: 85px;
+  left: 22px;
+}
+.main-fc {
+  width: 60%;
+}
+
+.other-videos {
+  width: 30%;
+}
+
+#autobutton {
+  border: none;
+  background-color: transparent;
+}
+.sidebar {
+  position: relative;
+  right: 35px;
+  top: 80px;
+}
+.image {
+  float: left;
+  height: 10px;
+  position: relative;
+}
+.aside-video {
+  grid-template-columns: 180px 180px;
+  display: grid;
+  margin-bottom: 4px;
+}
+.aside-description {
+  font-family: Roboto;
+  line-height: 1.2em;
```

Comment:
you are duplicating these line of codes too much of times try to give it it's own selector and uses it on your HTML instead